backend/api/elite2Api.js

```diff
@@ -148,6 +148,9 @@ const elite2ApiFactory = client => {
   const getMovementsIn = (context, agencyId, movementDate) =>
     get(context, `/api/movements/${agencyId}/in/${movementDate}`)
 
+  const getMovementsInBetween = (context, agencyId, params) =>
+    get(context, `/api/movements/${agencyId}/in?${mapToQueryString(params)}`)
```

Comment:
between?

Reply:
the endpoint this calls allows getting `movements in` across a range of dates, compared to the method above which just gets the `movements in` on a specific date.